Tests/Tests/SDWebImageTestCache.m

```diff
@@ -98,14 +128,37 @@ - (void)setData:(nullable NSData *)data forKey:(nonnull NSString *)key {
 }
 
 - (NSUInteger)totalCount {
-    return [self.fileManager contentsOfDirectoryAtPath:self.cachePath error:nil].count;
+    NSUInteger count = 0;
+    @autoreleasepool {
```

Comment:
Do we really need to call all the API for NSFileManager in autorelease pool ?

Reply:
With autorelease or not has no performance issues, but add autorelease can make the release time clear . 
Just like Defensive Programming.